tests/bgp_route_install_test.go

```diff
@@ -96,18 +96,18 @@ func routeInstallConfigureBGP(t *testing.T, dut *ondatra.DUTDevice) {
 func routeInstallConfigureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	t.Logf("Start Setting DUT Config")
 	routeInstallConfigureInterface(t, dut)
-	helpers.ConfigDUTs(map[string]string{"arista1": "../resources/dutconfig/bgp_route_install/set_dut.txt"})
+	helpers.ConfigDUTs(map[string]string{"arista": "../resources/dutconfig/bgp_route_install/set_dut.txt"})
```

Comment:
Can you remove the helpers.ConfigDUTs function and use the Ondatra Config API instead? This should work:
`dut.Config().New().
  WithAristaFile("../resources/dutconfig/bgp_route_install/set_dut.txt").
  Push(t)`

Reply:
done